packages/components/src/components/domainproperties/list/actions.ts

```diff
@@ -13,16 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ActionURL, Ajax, Utils, Domain } from '@labkey/api';
-
-import { buildURL } from '../../..';
+import { ActionURL, Ajax, Utils, Domain, getServerContext } from '@labkey/api';
 
 import { ListModel } from './models';
 
-export function getListProperties(listId?: number) {
+function getListProperties(listId?: number): Promise<ListModel> {
```

Comment:
Does `list-getListProperties.api` really not require a `listId` param? What does it do when the param is not supplied? Will the called just receive a Promise rejection in this case?

Reply:
Correct, it does not require a listId. In the case where it is empty, it will return the default properties for ListDomainKindProperties (see platform/list code in ListManger.java line 190). This is for the create list case to give the list properties panel default values based on the domain kind.

Reply:
Got it. Thanks for the follow up.